packages/style-minify/index.js

```diff
@@ -0,0 +1,13 @@
+const merge = require('deepmerge');
+const OptimizeCssPlugin = require('optimize-css-assets-webpack-plugin');
+
+module.exports = ({ config }, opts = {}) => {
+  const options = merge({
+    pluginId: 'style-minify',
+    plugin: {}
+  }, opts);
+
+  config
+    .plugin(options.pluginUseId)
```

Comment:
Need to update this to `pluginId` as well.

Reply:
(╯°□°）╯︵ ┻━┻

Reply:
lololol